src/tests/unit/common/assessment-store-data-builder.ts

```diff
@@ -5,12 +5,26 @@ import { AssessmentDataConverter } from '../../../background/assessment-data-con
 import { AssessmentStore } from '../../../background/stores/assessment-store';
 import { IAssessmentData, IAssessmentStoreData } from '../../../common/types/store-data/iassessment-result-data';
 import { VisualizationType } from '../../../common/types/visualization-type';
+import { InitialAssessmentStoreDataGenerator } from './../../../background/intial-assessment-store-data-generator';
 import { BaseDataBuilder } from './base-data-builder';
 
 export class AssessmentsStoreDataBuilder extends BaseDataBuilder<IAssessmentStoreData> {
-    constructor(provider: AssessmentsProvider, dataConverter: AssessmentDataConverter) {
+    constructor(
+        provider: AssessmentsProvider,
```

Comment:
I'm wondering if this should be private?

Reply:
I believe we do not keep it, so we don't need to do that 😄